src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/ComponentsWebAssembly-CSharp.Client.csproj.in

```diff
@@ -21,14 +21,10 @@
 
   <!--#endif -->
   <!--#if PWA -->
-  <ItemGroup>
-    <!-- When publishing, swap service-worker.published.js in place of service-worker.js -->
-    <Content Update="wwwroot\service-worker*.js" CopyToPublishDirectory="false" />
-    <ContentWithTargetPath Include="wwwroot\service-worker.published.js">
-      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
-      <TargetPath>wwwroot\service-worker.js</TargetPath>
-    </ContentWithTargetPath>
+  <!-- When publishing, swap service-worker.published.js in place of service-worker.js -->
+  <ItemGroup Condition="'$(DesignTimeBuild)' != 'true'">
```

Comment:
Should we add a comment that says why it's being done conditionally? Especially since this is in the user's project file.

Reply:
I can think of a good way to express this in a comment that would not take more space and would be confusing. I don't think anybody would care that much about it either. If you have suggestions I'm all ears.

Reply:
The existing comment does say "when publishing", so I would expect most developers to infer that the condition is to do with that. I'm fine with keeping the comment as-is. If we get questions, maybe we'll learn more about what people need to know and adjust accordingly.